maraboupy/test/test_nnet_extensions.py

```diff
@@ -0,0 +1,231 @@
+
+from maraboupy.MarabouNetworkNNet import *
+from maraboupy import Marabou
+
+TOL = 1e-8
+
+network_filename = "./resources/nnet/acasxu/ACASXU_experimental_v2a_1_9.nnet"
+
+layer = 2
+
```

Comment:
It's convention to write global variable constants using all uppercase letters (like "TOL" instead of "tol"), and this is the convention used in other test files. This also makes reading  and editing the code easier in the future.

Also, defining the path to the nnet file like this means you can only run the tests from the Marabou root folder, and you will get errors if you run the tests from a different folder. This can be remedied with os.path.join() and os.path.dirname(\_\_file\_\_). Please see the other test files as examples.

There are also a lot of extra empty lines in this file. Generally, a single empty line can be used to separate blocks of code or methods, and an empty line usually comes before a comment line, but there should only be a single empty line. The file should end with a single empty line, and the file should begin with a comment or line of code (not an empty line).

Reply:
Changes made.